src/app/shared/models/providerAdmin.model.ts

```diff
@@ -1,4 +1,5 @@
-export interface ProviderAdmin {
+
+export interface ProviderAdminBackend {
```

Comment:
it is not a good naming

Reply:
I don't really understand the difference between ProviderAdmin and ProviderAdmin Backend? Why do we have different models for that?

Reply:
ProviderAdminBackend is removed, thanks